src/components/PixiApp.tsx

```diff
@@ -71,6 +71,20 @@ const PixiApp: FunctionComponent<{}> = () => {
 						/>
 					);
 				}
+				if (TextM.is(entity)) {
+					const { id, params: { x, y, copy} } = entity;
+					
+					return (
+						<Text
```

Comment:
Here you can access isSelected and apply a different text style when it is. http://pixijs.download/release/docs/PIXI.Text.html

Something like
```jsx
style={isSelected ? { stroke: selectedColor } : {} }
```

SelectedColor is in config.js.

Reply:
(Hopefully) resolved